src/datatable/math.py

```diff
@@ -72,9 +71,33 @@
     trunc,
 )
 
+from datatable.lib._datatable import (
+    isna
+)
```

Comment:
The code above imports many functions from `from .lib._datatable`. Why do we need to have a separate import for `isna`?